corehq/apps/hqmedia/views.py

```diff
@@ -270,10 +272,101 @@ def update_multimedia_paths(request, domain, app_id):
             'error': _("Could not find file. Please re-upload file.")
         })
 
-    # TODO: update paths
+    app = get_app(domain, app_id)
+    from corehq.apps.hqmedia.view_helpers import validate_multimedia_paths_rows
+    with get_spreadsheet(f) as spreadsheet:
+        rows = list(spreadsheet.iter_rows())
+        (errors, warnings) = validate_multimedia_paths_rows(app, rows)
+        if len(errors):
+            return json_response({
+                'success': 1,
+                'errors': errors,
+            })
+
+        # Master list of old_path => new_path
+        paths = {row[0]: interpolate_media_path(row[1]) for row in rows}
```

Comment:
probably a miss in my knowledge but why can't we ask the user to give us path which does not need us to interpolate?

Reply:
It does throw a warning, but as long as we can interpolate for them, we might as well do it.

Reply:
if i am user and i am editing the paths, i would remove the extra `jr://` thing in beginning and then also update the path or i would just modify the path? i am assuming that the users start with the `jr://` path.

Reply:
As the user, you can do either. If you enter `path/to/thing`, `interpolate_media_path` will transform that to `jr://path/to/thing`. But if you enter `jr://path/to/thing`, `interpolate_media_path` won't do anything.